### PR TITLE
Pure python pipeline

### PR DESCRIPTION
**Pull request by sebbacon**
This is a new PR to integrate the changes proposed and discussed in #182.

It adds the minimal refactoring required to integrate into the pipeline and get the tests to pass, plus whitespace changes via `black`.

### CODE REVIEW

File: clinicaltrials/frontend/management/commands/load_data.py [hunk 1]
```diff
@@ -204,18 +213,448 @@ def process_data():
...
+            td["title"] = td["official_title"] or td["brief_title"]
+
+            if td["official_title"] is not None:
+                td["title"] = td["official_title"]
+            elif td["official_title"] is None and td["brief_title"] is not None:
+                td["title"] = td["brief_title"]
+            else:
+                td["title"] = None
+
+            # add this to "if" statement after testing: 'and completion_date is not null')
```
Comment (sebbacon) [inline clinicaltrials/frontend/management/commands/load_data.py]: @NickCEBM do you want to do this?

Comment (NickCEBM) [reply clinicaltrials/frontend/management/commands/load_data.py]: Have we confirmed that this code creates an identical CSV to the SQL yet?

Comment (NickCEBM) [reply clinicaltrials/frontend/management/commands/load_data.py]: Assume we have so added (knowing we can go back if not).

File: clinicaltrials/frontend/management/commands/load_data.py [hunk 2]
```diff
@@ -599,6 +598,7 @@ def convert_to_csv():
                     or primary_completion_date is None
                 )
                 and completion_date < date.today()
+                and completion_date is not null
```
Comment (sebbacon) [inline clinicaltrials/frontend/management/commands/load_data.py]: `None` in python - not `null`

Comment (NickCEBM) [reply clinicaltrials/frontend/management/commands/load_data.py]: d'oh
